spec/features/apply_for_help_with_fees_spec.rb

```diff
@@ -1,86 +1,108 @@
 # coding: utf-8
 require 'rails_helper'
 
+def find_button_by_label(label_text)
```

Comment:
It might have done.  Not sure why I did it this way now... if only I could remember what July Colin was thinking! :)